src/test/java/seedu/address/model/order/OrderDateTest.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.model.order;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import seedu.address.testutil.Assert;
+
+public class OrderDateTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        Assert.assertThrows(NullPointerException.class, () -> new OrderDate(null));
+    }
+
+    @Test
+    public void constructor_invalidName_throwsIllegalArgumentException() {
+        String invalidDate = "12/10/18";
+        Assert.assertThrows(IllegalArgumentException.class, () -> new OrderDate(invalidDate));
```

Comment:
Should it throw ParseException instead?

Reply:
This one has been checked already (the ParseException throws the IllegalArgument)